src/components/Toolbar.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <ion-header :translucent="true">
+    <ion-toolbar>
+      <div id="title-bar" >
+        <ion-title>Product Order</ion-title>
+        <ion-title id="middle-title">{{titleText}}</ion-title>
+      </div>
+    </ion-toolbar>
+  </ion-header>
+</template>
+
+<script language="ts">
+import {IonTitle, IonButton, IonHeader, IonToolbar} from '@ionic/vue';
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "Toolbar",
+  components: {
+    IonTitle,
+    IonHeader,
+    IonToolbar
+  },
+  props: ['titleText'],
```

Comment:
```suggestion
  props: {
    titleText: {
      type: String,
      required: true
    },
  },
```

Always use the full form. See [style guide for prop definitions](https://v3.vuejs.org/style-guide/#prop-definitions-essential).